free-2.md

```diff
@@ -11,34 +11,41 @@ _What need is met by your idea? Who are you helping? What is that person's
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would require the students to invest time in setting up their calendars but it would give them the flexibility, ease, and detailed control needed to put in all their due dates at the beginnning of the semester and then add additional pieces as they go. They could more easily express their busy lives in a calendar using this DSL than in the current, widely used software. A DSL is the appropriate tool, because this is a very limited, specific domain, but we want users to be able to easily express anything within it.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This seems so relevant to the lives of all college students. I have often felt frustrated with the difficulty of specifying exactly what I want on a calendar or of needing to use multiple tools to keep track of everything I am doing throughout the day. I was thinking about it some before I looked at any examples of past projects and was encouraged in my idea when I read about what NCarter3 did for his DSL in 2014 and what aputman did for his project in 2015.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Organizing tasks and activities together.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I would like to allow the user to type a command perhaps in terminal adding a new event or task and for them to immediantly see the updated result in their calendar. This would allow them to easily add new things to their calendar and if I create a simple documentation of what commands are available they can look up what kind of thing they want to add, type it in and then instantly see their result.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When a program runs a new event or activity should be added to the calendar and become visible to the user. Their might be errors if the user tries to use an invalid date or the parser does not understand an input and the program should return an error message with the word it cannot interpret or the invalid date.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to make a daily todo list, to carry events over from a previous day, to schedule appointments, including ones with complicated repitition patterns and set reminders. There shouldn't be very many things that are possible but difficult and it should be impossible to create a program that does anything beyond editing a calendar.
 
```

Comment:
I don't think I quite visualize how the todo list fits in with the calendar--is the assumption that you have deadlines on all your todo items?